cmake/OpenCVPackaging.cmake

```diff
@@ -246,6 +246,10 @@ endif()
 set(DEV_LINTIAN_OVERRIDES "binary-or-shlib-defines-rpath" # usr/bin/opencv_traincascade
                           "binary-without-manpage") # usr/bin/opencv_traincascade
 
+if(AARCH64)
```

Comment:
It's not a portable solution too. The same package can be checked with different versions of lintian and it will lead to errors.